User object omitted from integration creates (#5783)
hammerandchisel/discord-api-docs: docs/topics/Gateway_Events.md

#### Integration Create

> Sent when an integration is created. The inner payload is an [integration](#DOCS_RESOURCES_GUILD/integration-object) object with an additional `guild_id` key, however the `user` object is omitted from the event

Reviewer: ```suggestion
Sent when an integration is created. The inner payload is an [integration](#DOCS_RESOURCES_GUILD/integration-object) object with `user` omitted and an additional `guild_id` key:
```
